decidim-accountability/app/views/decidim/accountability/admin/results/_form.html.erb

```diff
@@ -54,11 +54,40 @@
 
     <div class="row column">
       <% if @form.proposals %>
-        <%= form.select :proposal_ids,
-                        @form.proposals,
-                        {},
-                        { multiple: true, class: "chosen-select" }
-        %>
+<!-- dataPicker ------------------------------------------------- -->
```

Comment:
If we need a comment to highlight this part, then I'd move this into a partial (and remove the comment)

Reply:
After extracting js there is no need for the comment and the block that remains seems small enough to be kept here.